poisson2d/README.md

```diff
@@ -0,0 +1,31 @@
+## Poisson solver in various languages
+
+This solver uses a simple Jacobi iteration to solve a Poisson equation. For details, see, for example,
+"Computational Physics" by Mark Newman, Chap 9. Compiler commands were:
+
+```gfortran sourcefile.f95 -Ofast -o program```
+
+```gcc sourcefile.c -Ofast -o program```
+
+For Cython module (import it to run):
+
+```python setup.py build_ext --inplace```
+
+And the timing results (the grid is MxM):
+
+| Language            | M=100 (seconds) | M=200 (seconds)       | M=300 (seconds)        |
+|---------------------|-----------------|-----------------------|------------------------|
+| Python (pure)       | 276             | Until sun's expansion | Heat death of universe |
```

Comment:
I appreciate the humor, but I suggest just using "n/a" or similar, other communities could receive the original wording as ridiculing. And it's not strictly correct.

```suggestion
| Python (pure)       | 276             | n/a                   | n/a                    |
```

Reply:
I think it's a good joke, but it gets old when you have to read it often, so `n/a` seems better.

Reply:
Replaced those two with n/a